### PR TITLE
Add support for "x-special/gnome-copied-files" on Linux

### PR DESCRIPTION
**Pull request by daschuer**
This allows to use Crtl+C and Ctrl+V with various Linux file managers

### CODE REVIEW

File: src/util/clipboard.cpp [hunk 1]
```diff
@@ -15,6 +31,11 @@ void Clipboard::add(const QUrl& url) {
 void Clipboard::finish() {
     QMimeData* data = new QMimeData;
     data->setUrls(instance()->m_urls);
+#ifdef __LINUX__
```
Comment (1fabi0) [inline src/util/clipboard.cpp:34]: The limitation on linux is a bit confusing here because it depends on the file manager not on the OS

Comment (daschuer) [reply src/util/clipboard.cpp:34]: It would not break anything to make it available for all targets. But is there a real use case for it?

Comment (1fabi0) [reply src/util/clipboard.cpp:34]: For me it's not about the real use case, it's more about the difference between the OS targets caused by a file-manager difference that could in theory exist on all platforms

**Comment by m0dB** [comment]
Thanks for this, @daschuer !

**Comment by daschuer** [comment]
Oh Sorry, I see I have pushed directly to your branch without noticing. I need more careful next time.

**Pull request merged**